manager/src/main/java/io/gingersnapproject/search/SearchResult.java

```diff
@@ -0,0 +1,17 @@
+package io.gingersnapproject.search;
+
+import java.util.List;
+
+public interface SearchResult {
+
+   long hitCount();
+
+   boolean hitCountExact();
+
+   List<SearchHit> hits();
+
+   boolean hitsExact();
```

Comment:
`hitsExact` is about the accuracy of the hits, this is always true if no timeout occurs. In case of timeout Elasticsearch (or Hibernate Seerch) returns a partial list of hits and `hitsExact` will be `false`.
While `hitCountExact` is about the accuracy of the `hitCount`.
You can find more about that on [query result object topic](https://infinispan.zulipchat.com/#narrow/stream/313207-memory-service/topic/query.20result.20object).